src/Routes/RosPage/RosPage.js

```diff
@@ -240,6 +240,7 @@ class RosPage extends React.Component {
                                     {
                                         label: SFObject.label,
                                         type: SFObject.type,
+                                        value: `checkbox-state`,
```

Comment:
@karelhala @kgaikwad Based on the code here https://github.com/RedHatInsights/frontend-components/blob/master/packages/components/src/ConditionalFilter/ConditionalFilter.js#L133 and https://github.com/RedHatInsights/frontend-components/blob/master/packages/components/src/ConditionalFilter/ConditionalFilter.js#L62 and referring others frontend repo I made this change.
Let me know if you have suggestion on better approach for same.

Reply:
Try removing the value from `filterValues` and add it here
```suggestion
                                        value: this.state.stateFilterValue,
```

Reply:
@karelhala I tried this but with this change as soon as I select the state filter from the list the dropdown changes to Name again. Here is the screenshot for your reference
![image](https://user-images.githubusercontent.com/5928530/148835581-425c54d6-b29a-4042-a477-1647e3f1f0c8.png)

Reply:
Aaah yeah! I thought you were setting the value for the whole conditional filter. This is the right approach and works as should.